omod/src/main/resources/config.xml

```diff
@@ -54,6 +54,14 @@
 		<require_module version="${metadatamappingVersion}">
 			org.openmrs.module.metadatamapping
 		</require_module>
+
+		<require_module version="${reportingVersion}">
+			org.openmrs.module.reporting
+		</require_module>
+
+		<require_module version="${metadatadeployVersion}">
+			org.openmrs.module.metadatadeploy
+		</require_module>
```

Comment:
Yes, without this particular, there is a runtime error when I deploy the referenceApplication module. I had a chat about this with Daniel on IRC. However I will try to run the module by removing this dependency. If it doesn't, I'll keep this.